spec/api/labwares_spec.rb

```diff
@@ -1,40 +1,96 @@
 # frozen_string_literal: true
 
 # Retrieve location information for a given piece of labware (GET /api/labwares/<barcode>) returns labware object and location object
+# Retrieve labwares for a given list of locations barcodes (GET /api/labwares?location_barcodes=<barcode_1>,<barcode_2>...)
 
 require "rails_helper"
 
 RSpec.describe Api::LabwaresController, type: :request do
   let!(:location) { create(:location_with_parent) }
   let!(:labware) { create(:labware_with_audits, location: location) }
 
-  before(:each) do
-    get api_labware_path(labware.barcode)
-    @json = ActiveSupport::JSON.decode(response.body)
-  end
+  describe '#show' do
+    before(:each) do
+      get api_labware_path(labware.barcode)
+      @json = ActiveSupport::JSON.decode(response.body)
+    end
 
-  it "should be a success" do
-    expect(response).to be_successful
-  end
+    it "should be a success" do
+      expect(response).to be_successful
+    end
 
-  it "should return labware barcode" do
-    expect(@json["barcode"]).to eq(labware.barcode)
-  end
+    it "should return labware barcode" do
+      expect(@json["barcode"]).to eq(labware.barcode)
+    end
 
-  it "should return location object associated with labware" do
-    expect(@json["location"]).to_not be_empty
-  end
+    it "should return location object associated with labware" do
+      expect(@json["location"]).to_not be_empty
+    end
 
-  it "should return link to labware audits" do
-    expect(@json["audits"]).to eq(api_labware_audits_path(labware.barcode))
+    it "should return link to labware audits" do
```

Comment:
Just wondering whether we need to return the audits? We definitely don't need them for this call and I don't know whether anyone else is using it in this context. We will be moving away from audits and putting it in the warehouse instead.

Reply:
Oh right, I didn't know about the change to putting this info in the warehouse. This test isn't a new one - I just transferred it into a 'describe' block. It's to do with the existing endpoint.

However, I think the new 'index' endpoint will also return the audit link, but it's not something I intentionally put in, and I'd have to figure out how to get rid of it. Do you think it's worth it?

Reply:
Ah ok. Sorry I misunderstood. Didn't read properly. I thought we were returning the audits with the barcodes. Just leave it. Removing the audits is part of another story.